### DISCUSSION TITLE
Connecting to Frigate from the HA proxy add-on over https

### DISCUSSION THREAD

**Discussion by calebcall** [blakeblackshear/frigate#5838] [Ask A Question]
I have Frigate successfully running on my k8s cluster.  I have that setup at https://frigate.mydomain.com.  I'm trying to get the HA Proxy add-on to connect and it's unable to.  If I put the server address as `frigate.mydomain.com:443` it then tries to connect as http://frigate.mydomain.com:443 and gets a connection refused because it's sending http traffic over an https port.  If I set the server as `https://frigate.mydomain.com` then it registers as invalid as the regex is looking for the port at the end `:\d+`.  If I try to set the server to `https://frigate.mydomain.com:443` it saves but then throws the error: `nginx: [emerg] invalid host in upstream "https://frigate.mydomain.com:443" in /etc/nginx/includes/upstream.conf:2`

So is there a way to connect to frigate from the proxy add-on over https?

**Comment by antonjs** [comment]
Just ran into this same problem; would love to know if there's a way to do it without exposing Frigate over HTTP.
